carson/slack/client.py

```diff
@@ -0,0 +1,35 @@
+import requests
+from .. import app
+
+
+class SlackClient(object):
```

Comment:
i copied a lot of this out of `slack/actions` to be a more self-contained module, and I couldn't quite think of an easy way to do the Appointmenting from the test failure message